contrib/mesos/pkg/scheduler/service/service_test.go

```diff
@@ -124,42 +124,3 @@ func Test_DefaultResourceLimits(t *testing.T) {
 	assert.Equal(s.DefaultContainerCPULimit, mresource.DefaultDefaultContainerCPULimit)
 	assert.Equal(s.DefaultContainerMemLimit, mresource.DefaultDefaultContainerMemLimit)
 }
-
-func Test_StaticPods(t *testing.T) {
```

Comment:
this isn't doing anything static-pod specific, it's just testing the archive package. the archive package already has unit tests